library/MedEx/API.php

```diff
@@ -950,13 +1024,31 @@ public function generate($token, $events)
                                         ".$frequency."
                                         ".$no_dupes."
                                     ORDER BY cal.pc_eventDate,cal.pc_startTime";
-                $result = sqlStatement($sql_GOGREEN, $escapedArr);
-
+              //  $this->MedEx->logging->log_this($sql_GOGREEN);
+              //  $this->MedEx->logging->log_this($escapedArr);
+                try {
+                    $result = sqlStatement($sql_GOGREEN, $escapedArr);
+                } catch (\Exception $e) {
+                     $dd = print_r($escapedArr, true);
+                    echo "hello<pre>$sql_GOGREEN\n".$dd."\n".$e."\n";
```

Comment:
I'm assuming above is a debug thing and should be commented out

Reply:
Yup.
Fix coming...